libraries/networking/src/UserActivityLogger.h

```diff
@@ -55,6 +59,7 @@ private slots:
 private:
     UserActivityLogger();
     Setting::Handle<bool> _disabled { "UserActivityLoggerDisabled", true };
+    Setting::Handle<bool> _crashDisabled { "CrashLoggerDisabled", false };
```

Comment:
can this be renamed to _crashReporterEnabled ?  ,  and option CrashReporterEnabled

Reply:
or Reporting